src/components/views/sul-people/sul-people-table-view.tsx

```diff
@@ -53,8 +53,10 @@ const SulPeopleTableView = ({items, hasHeading}: Props) => {
       const existingIndex = newState.indexOf(type)
       if (existingIndex >= 0) {
         newState.splice(existingIndex, 1)
+        console.log("newState splice line")
```

Comment:
Hi @pookmish I want to change the toggle behavior for the specialist. The ticket says "Toggle should only allow for one selection (like radio button group behavior)". I changed the buttons from checkboxes to radio and removed the group hover because it seemed to confuse thing.  I added console logs and it doesn't seem to  go to the splice part of the if. and the link 59 logs out two comments per click. 

Can you point me in the correct direction?

Reply:
Keep in mind that if we change them to radio buttons, we will have to show the "All Specialists" on mobile as well.

Reply:
Yes, they want to have the `All` there now too. They changed it to just All.

Reply:
Ok. Would you like me to update this PR using radios?

Reply:
I was reading an article and trying to get it to work. Let me see if I figure it out.

Reply:
Ok, can you update the PR with radios?

Reply:
I've updated the PR. See the last commit to familiarize yourself with this approach. It's more simple than the checkboxes.
Note that you will need to have some "checked" states. Use `peer-checked:....` on the `<span>` tags immediately after the `<input>` tags.

Reply:
That was helpful to see.  It is a lot simpler.